docs/TranslationGuide.md

```diff
@@ -1,49 +1,175 @@
-#To translate the Zonemaster frontend interface 2 files must be added.
+# Translation Guide
 
-* The JSON structure holding all the messages of the Web Interface
-* The file holding the FAQ
+This guide gives instructions for how to add a new language to
+Zonemaster-GUI. The language is assumed to exist in or to be added
+to Zonemaster-Engine and Zonemaster-Backend.
 
-1. Adding the JSON structure holding all the messages of the Web Interface
+When updating a language in use, this document could also be used as
+a checklist for where the changes are done.
 
-The file is located in the following folder:
-```
-ZONEMASTER_DISTRIBUTION/zonemaster-gui/src/assets/i18n
-```
-This folder contains the language files.
+You should usually read this document from develop branch to make
+sure you have the latest version.
 
-Each language file contains a hash structure with English/Default messages as keys and the translated messges as values.
-The file should be named with the [official language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) and must have .json extension.
+## Language code
 
-2. Adding the file holding the FAQ
-The file is located in the following folder:
-```
-ZONEMASTER_DISTRIBUTION/zonemaster/docs/FAQ
-```
-The filename must be of the format: qui-faq-LANGUAGE_CODE.md
-Where LANGUAGE_CODE is the standard 2 letter language code from: [official language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
+Zonemaster uses [ISO 639-1] two-letter language codes, normally in
+lower case, but in GUI sometimes with first letter in upper case
+to make the display nicer. GUI is currently available in the
+following languages with the attached language code:
+
+* `da` for Danish language
+* `en` for English language
+* `fr` for French language
+* `sv` for Swedish language
+
+
+## Submitting changes
+
+Below are instructions for how to add or modify files. Preferably,
+submit the new or updated file as a pull request to Github (see
+[translators guide for Engine]). Contact the Zonemaster Group if
+that does not work.
+
+The changes in or addition of *LANG.json* and *gui-faq-LANG.md* must
```

Comment:
`in or` : I think there are too many words

Reply:
I'll provide an update.